pkg/services/stats/statsimpl/stats.go

```diff
@@ -173,6 +178,11 @@ func (ss *sqlStatsService) GetAdminStats(ctx context.Context, query *stats.GetAd
 		dailyActiveEndDate := now.Add(-dailyActiveUserTimeLimit)
 		monthlyActiveEndDate := time.Date(now.Year(), now.Month(), 1, 0, 0, 0, 0, now.Location())
 
+		alertsQuery := fmt.Sprintf("SELECT COUNT(*) FROM %s", dialect.Quote("alert"))
+		if ss.IsUnifiedAlertingEnabled() {
+			alertsQuery = fmt.Sprintf("SELECT COUNT(*) FROM %s", dialect.Quote("alert_rule"))
+		}
```

Comment:
I'm not proud of this... and I can probably change it to use xorm semantics, but:

A) I don't know how frequent this area sees changes - probably not that many given this has been broken for so long.
B) I'm not a fan of reactors for no reason

Reply:
I'm sorry - I don't think we should do that. I'm not sure who the owner of this area is, and this will require testing on other databases to ensure that it works correctly. 

Seems a lot of work for little value, given I don't see any evidence of performance problems in this area.

Reply:
If you feel too strongly about this - I'd say we go the full mile and refactor this with xorm so that we can have a bit more confidence that the ORM will protect us.

Reply:
From the context surrounding this PR, I suggest we do not invest more time into this.